README.md

```diff
@@ -20,6 +20,78 @@ ArgoCD, in its default configuration, would consider this application "healthy."
 ## Our Solution
 To address this challenge, we've developed a solution using Lua scripts, which can detect and report the health status of custom workload resources. These Lua scripts, along with corresponding tests, are intended to be contributed back to the ArgoCD project. By doing so, we aim to empower ArgoCD with the ability to recognize and display the health status of advanced workload resources accurately.
 
+## How to use the integration scripts with Argo-cd 
+
+This README will guide you through the process of integrating Lua scripts with ArgoCD for managing workloads. Lua scripts can help ArgoCD monitor the health status of workloads effectively. In this example, we'll use the following CloneSet:
+
+### Step 1: Install Openkruise as per the instructions given [here](https://openkruise.io/docs/installation/)
+
+### Step 2: Setup the Argo-cd Pipeline or follow [this](https://openkruise.io/docs/best-practices/gitops-with-kruise/#tekton-pipeline--argo-cd) guide 
+
+1. Clone the repo [kruise-eg](https://github.com/maheshkasabe/kruise-eg) and kubectl apply -f argocd.yaml
+
+2. You will have an argocd pipeline setup ready to apply and test artifacts
+
+3. Change the image version or replicas count inside openkruise_workloads/cloneset/cloneset.yaml 
+
+4. Now you will see the changes reflected inside your kubernetes container
+
+### Step 3: Argo-cd CloneSet Health Check
+
+Configure CloneSet Argo-cd [Custom CRD Health Checks](https://argo-cd.readthedocs.io/en/stable/operator-manual/health/#custom-health-checks), With this configuration argo-cd is able to perform a healthy check of the CloneSet, such as whether the CloneSet is published and whether the Pods are ready, as follows：
```

Comment:
can we provide a script to generate argo integration configmap from the luascript in kruise-argo repository